docs-en/chapter_preface/about_the_book.md

```diff
@@ -1,45 +1,45 @@
 # The Book
 
-The aim of this project is to create an open source, free, novice-friendly introductory tutorial on data structures and algorithms.
+The aim of this project is to create an open source, free, beginner-friendly crash course on data structures and algorithms.
```

Comment:
The aim of this project is to create an open source, free,
|
This open-source project aims to create a free,

Reply:
Sorry this is a typo, will fix it